app/models/proposal.rb

```diff
@@ -30,6 +30,8 @@ class Proposal < ActiveRecord::Base
 
   acts_as_taggable
 
+  # we do not have "dependent:" declaration on "steps" because there can be circular references
```

Comment:
Writing a test to illustrate why *not* to define something seems hard (impossible?) to me. It would require 2 different association definitions to compare the different behavior, yes?

How about we just eliminate the comment?